shenyu-spring-boot-starter/shenyu-spring-boot-starter-gateway/src/main/java/org/apache/shenyu/springboot/starter/netty/ShenyuNettyWebServerConfiguration.java

```diff
@@ -54,42 +54,42 @@ public NettyTcpConfig nettyTcpConfig() {
      * @return the netty reactive web server factory
      */
     @Bean
-    public NettyReactiveWebServerFactory nettyReactiveWebServerFactory(final ObjectProvider<NettyTcpConfig> config) {
+    public NettyReactiveWebServerFactory nettyReactiveWebServerFactory(final ObjectProvider<NettyTcpConfigurationProperties> config) {
         NettyReactiveWebServerFactory webServerFactory = new NettyReactiveWebServerFactory();
-        NettyTcpConfig nettyTcpConfig = Optional.ofNullable(config.getIfAvailable()).orElse(new NettyTcpConfig());
-        webServerFactory.addServerCustomizers(new EventLoopNettyCustomizer(nettyTcpConfig));
+        NettyTcpConfigurationProperties nettyTcpConfigurationProperties = Optional.ofNullable(config.getIfAvailable()).orElse(new NettyTcpConfigurationProperties());
+        webServerFactory.addServerCustomizers(new EventLoopNettyCustomizer(nettyTcpConfigurationProperties));
         return webServerFactory;
     }
 
     private static class EventLoopNettyCustomizer implements NettyServerCustomizer {
 
-        private final NettyTcpConfig nettyTcpConfig;
+        private final NettyTcpConfigurationProperties nettyTcpConfigurationProperties;
```

Comment:
nettyTcpConfigurationProperties  replac to  properties

Reply:
rename to NettyTcpProperties. done!